src/opm/parser/eclipse/EclipseState/SummaryConfig/SummaryConfig.cpp

```diff
@@ -795,6 +795,23 @@ inline void handleKW( SummaryConfig::keyword_list& list,
 
 // =====================================================================
 
+SummaryNode::SummaryNode(const std::string& kword,
+                         Category cat,
+                         const Location& locat,
+                         Type type,
+                         const std::string& name,
+                         int number,
+                         bool udef)
+    : keyword_(kword)
+    , category_(cat)
+    , loc(locat)
+    , type_(type)
+    , name_(name)
+    , number_(number)
+    , userDefined_(udef)
+{
+}
+
```

Comment:
Technically you could achieve the same effect using the functionality already in place:
```C++
auto snode = SummaryNode{ kword, cat, locat }
    .parameterType(type)
    .namedEntity(name)
    .number(number)
    .isUserDefined(udef);
```

Reply:
aha, i did not notice the setters (usually not there). sure, i can use those instead if you prefer.